integration_tests/snaps/asset-tracking/snapcraft.yaml

```diff
@@ -10,3 +10,39 @@ parts:
     plugin: nil
     stage-packages: ['hello=2.10-1']
     build-packages: ['hello=2.10-1']
+  git-part:
+    plugin: nil
+    source: git-source
+    source-type: git
+  git-part-branch:
+    plugin: nil
+    source: git-source
+    source-type: git
+    source-branch: feature
+  git-part-tag:
+    plugin: nil
+    source: git-source
+    source-type: git
+    source-tag: feature-tag
+  bzr-part:
+    plugin: nil
+    source: bzr-source
+    source-type: bzr
+  bzr-part-tag:
+    plugin: nil
+    source: bzr-source
+    source-type: bzr
+    source-tag: feature-tag
+  svn-part:
+    plugin: nil
+    source: svn-source
+    source-type: subversion
+  hg-part:
+    plugin: nil
+    source: hg-source
+    source-type: mercurial
+  hg-part-tag:
+    plugin: nil
+    source: hg-source
+    source-type: mercurial
+    source-tag: feature-tag
```

Comment:
This is a big snap. Which is not so bad, but what would you think about making one for each source? asset-tracking-with-git, asset-tracking-with-hg, asset-tracking-with-bzr...